lib/timeline/Core.js

```diff
@@ -657,17 +658,28 @@ Core.prototype.getDataRange = function() {
  *                                    provided to specify duration and easing function.
  *                                    Default duration is 500 ms, and default easing
  *                                    function is 'easeInOutQuad'.
+ * @param {Function} a callback funtion to be executed at the end of this function 
  */
-Core.prototype.setWindow = function(start, end, options) {
+Core.prototype.setWindow = function(start, end, options, callback) {
+  if (typeof arguments[2] == "function") {
+    callback = arguments[2]
+    options = {};
+  }
   var animation;
   if (arguments.length == 1) {
     var range = arguments[0];
     animation = (range.animation !== undefined) ? range.animation : true;
-    this.range.setRange(range.start, range.end, animation);
+    this.range.setRange(range.start, range.end, { animation: animation });
+  }
+  else if (arguments.length == 2 && typeof arguments[1] == "function") {
+    var range = arguments[0];
+    callback = arguments[1];
+    animation = (range.animation !== undefined) ? range.animation : true;
+    this.range.setRange(range.start, range.end, { animation: animation }, callback);
   }
   else {
     animation = (options && options.animation !== undefined) ? options.animation : true;
-    this.range.setRange(start, end, animation);
+    this.range.setRange(start, end, { animation: animation }, callback);
```

Comment:
Short answer: No.
I really think the callback function should be separated from the options. 
It's not exactly an option and should be passed through in an object.